08-writing-educational-content.html.md

```diff
@@ -0,0 +1,83 @@
+---
+title: How to Write Educational Content
+layout: article
+---
+
+At Mailchimp, educational content appears within our [Help Center](https://mailchimp.com/help/), as well as within our [Help Center](https://mailchimp.com/workshops/), certain marketing materials, and internal training documents. This section lays out the guiding principles for writing educational content, and discusses the main types of educational content that exist at Mailchimp.
```

Comment:
second link here I think should be `[workshops](https://mailchimp.com/workshops/)`

Reply:
Woops! Thanks for the catch! 🙏 
Fixed!